development/server_requests.py

```diff
@@ -15,3 +14,14 @@ def send_challenge(*args, **kwargs):
         '{}:{}/challenge'.format(SERVER_URL, SERVER_PORT),
         json=data,
     )
+
+
+def get_logs(*args, **kwargs):
```

Comment:
Why not passing explicit arguments like `get_logs(game_id, page_token, requests)` ?
If someone wants to use this function he will not know what this function expects

Reply:
done!